package.json

```diff
@@ -13,7 +13,8 @@
     "build": "parallel-webpack && npm run bundlewatch",
     "bundlewatch": "bundlewatch --config ./bundlewatch.config.js",
     "doc": "jsdoc -c jsdoc-conf.json",
-    "test": "karma start --single-run --browsers=ChromeHeadless",
+    "pretest": "npm run build && npm run copybuilds",
```

Comment:
@maoznir , is the pretest part of this change?

Reply:
yes

Reply:
Why is that change requires a pretest?

Reply:
I've refactored:
1. remove pretest
2. update test script to test folder structure
3. update prepublish script to verify packages, where it's actually needed

Reply:
refactored again due to travis breaking, please see updated changes